package.json

```diff
@@ -18,11 +18,11 @@
     "@fortawesome/react-fontawesome": "^0.1.8",
     "@rails/webpacker": "4.2.2",
     "@reach/router": "^1.3.3",
+    "@single-cell-portal/igv": "2.6.4-beta.1",
```

Comment:
we'd decided to alphabetize this under 'i' for igv, so it can be tracked in the same place as the igv dependency.  But if it's making it too much of a hassle to add dependencies (since it will always complain about them not being sorted), I'm ok putting it here

Reply:
Same.  I suspect the old placement and its now one are both the result of slightly different tools.

Reply:
This is likely an artifact of running `yarn add` and `yarn remove` several times as I was trying out different `swagger-ui` packages.  I'll fix that.

Reply:
300fdaf